### PR TITLE
Custom table names support

### PR DESCRIPTION
**Pull request by EinKrebs**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -17,9 +17,9 @@ password:
 database:
 # Endpoint for scraping prometheus metrics. Default localhost:9090.
 metrics_endpoint: localhost:9090
-# Table with spans. Default "jaeger_spans_local".
+# Table with spans. Default "jaeger_spans".
```
Comment (pavolloffay) [inline config.yaml:20]: Why are we renaming the names to `jaeger_spans`? The embedded scripts create "local" tables and it seems to be convention to use `_local` suffix for these tables.

File: embed.go [hunk 2]
```diff
@@ -2,5 +2,8 @@ package jaegerclickhouse
 
 import "embed"
 
-//go:embed sqlscripts/*
-var EmbeddedFiles embed.FS
+//go:embed sqlscripts/no_replication/*
+var EmbeddedFilesNoReplication embed.FS
```
Comment (pavolloffay) [inline embed.go:6]: Can you explain why are you moving the scripts? I don't see why it is needed.

File: storage/store.go [hunk 3]
```diff
@@ -30,13 +30,29 @@ type Store struct {
 	archiveReader spanstore.Reader
 }
 
+// Represents query with C-style placeholders and params for this placeholders
+type customStmt struct {
```
Comment (pavolloffay) [inline storage/store.go:34]: Is this used only for the embedded scripts? If so I would prefer to omit this type and use`fmt.Sprintf` directly for the substitution.

File: embed.go (hunk 2 above)
Comment (EinKrebs) [reply embed.go:6]: Yeah, it's not needed here, gonna revert this

File: config.yaml (hunk 1 above)
Comment (EinKrebs) [reply config.yaml:20]: Is the suffix needed if we don't use any "global" tables? I thought to add the suffix to local tables automatically if and only if we use distributed table.

Comment (pavolloffay) [reply config.yaml:20]: I think it's better to be explicit and use always `_local` for local tables. It's clear what it means and how it behaves.

Comment (EinKrebs) [reply config.yaml:20]: Okay, so be it

File: sqlscripts/0003-jaeger-operations.sql [hunk 4]
```diff
@@ -8,5 +8,5 @@ AS SELECT
     service,
     operation,
     count() as count
-FROM jaeger_index_local
+FROM %s
```
Comment (pavolloffay) [inline sqlscripts/0003-jaeger-operations.sql:11]: Is it possible to add a comment to this file. I would like to mention here that here goes `jaeger_index_local`

File: config.yaml (hunk 1 above)
Comment (pavolloffay) [reply config.yaml:20]: then please revert this file

**Comment by pavolloffay** [comment]
could you please rebase to get the latest main changes here (e2e test)

**Comment by EinKrebs** [comment]
Done it already

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**